<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StimConfigUnit</class>
 <widget class="QWidget" name="StimConfigUnit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>28</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item row="0" column="1">
    <widget class="QComboBox" name="wvfmBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="Loc1Box"/>
   </item>
   <item row="0" column="3">
    <widget class="QComboBox" name="Loc2Box"/>
   </item>
   <item row="0" column="4">
    <widget class="QCheckBox" name="enStim">
     <property name="text">
      <string>Enable</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="stimLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
